core/src/main/java/com/exadel/aem/toolkit/core/injectors/annotations/RequestSuffix.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.core.injectors.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apache.sling.models.annotations.Source;
+import org.apache.sling.models.spi.injectorspecific.InjectAnnotation;
+
+import com.exadel.aem.toolkit.core.injectors.RequestSuffixInjector;
+
+/**
+ * Annotation can be used on either methods, fields or parameters to let Sling Models inject a request
+ * suffix from the RequestPathInfo into fields of type String, Resource or Object, if type is not supported returns null.
+ */
+@Target({ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER})
```

Comment:
I don't see any examples of tests where the injector is used on the method. can we add these test cases?

Reply:
Yes, of course we can.